docs/sources/docker-hub/accounts.md

```diff
@@ -34,21 +34,52 @@ page.
 
 ## Organizations and groups
 
-Also available on the Docker Hub are organizations and groups that allow
-you to collaborate across your organization or team. You can see what
-organizations [you belong to and add new organizations](
+To help you collaborate across your projects or teams, Docker Hub also
+provides organizations and groups that allow more than one Hub user to work on
+shares repositories.
+
+### Creating and viewing organizations
+
+You can see what organizations [you belong to and add new organizations](
 https://hub.docker.com/account/organizations/) from the Account Settings
-tab. They are also listed below your user name on your repositories page and in your account profile.
+tab. They are also listed below your user name on your repositories page
+and in your account profile.
 
 ![organizations](/docker-hub/hub-images/orgs.png)
 
-From within your organizations you can create groups that allow you to
-further manage who can interact with your repositories.
+### Organization groups
+
+Users that are in the `Owners` group for an organization, can create and
+modify the membership of groups.
+
+Unless they are the organization's `Owner`, users can only see groups that they
+are members of.
 
 ![groups](/docker-hub/hub-images/groups.png)
 
-You can add or invite users to join groups by clicking on the organization and then clicking the edit button for the group to which you want to add members. Enter a user-name (for current Hub users) or email address (if they are not yet Hub users) for the person you want to invite. They will receive an email invitation to join the group.
+### Repository group permissions
+
+Organization groups can then be used to manage who can interact with your
```

Comment:
This whole section needs to be written largely in the active voice. 
![image](https://cloud.githubusercontent.com/assets/1347209/7415459/e19c8a32-ef0d-11e4-8195-8734a92041a9.png)

For example, the first sentence 

"Use organization groups to manage the users that interact with your repositories."